R/twas.R

```diff
@@ -130,13 +152,14 @@ twas_weights_cv <- function(X, Y, fold = NULL, sample_partitions = NULL, weight_
                 args <- weight_methods[[method]]
                 if (method %in% multivariate_weight_methods) {
                     # Apply multivariate method to entire Y for this fold
-                    weights_matrix <- do.call(method, c(list(X = X_train, Y = Y_train), args))
+                    weights_matrix <- do.call(method, c(list(X = X_train, Y = Y_train, sample_partition=sample_partition), args))
```

Comment:
why do you still need sample_partition in the method itself?

Reply:
It's now been removed